sdks/java/core/src/main/java/org/apache/beam/sdk/metrics/MetricFiltering.java

```diff
@@ -19,6 +19,7 @@
 
 import java.util.Set;
 import org.apache.beam.vendor.guava.v26_0_jre.com.google.common.base.Objects;
+import org.checkerframework.checker.nullness.qual.Nullable;
```

Comment:
`javax.annotations.Nullable` has incorrect qualifiers, so it cannot be applied in all contexts where it is applicable.